### PR TITLE
Use an arm64 runner for CI

### PR DESCRIPTION
**Pull request by hannesa2**
Let's see where the issues are 
image

### CODE REVIEW

**Comment by hannesa2** [comment]
@DivineDominion 
Now it builds on arm64 as well.

Next upcoming thing could be a release pipeline, with an autogenerated changelog

File: .github/workflows/BuildPR.yml [hunk 1]
```diff
@@ -15,10 +15,13 @@ jobs:
...
-#          - xcode: Xcode
-#            os: ARM64
-#            architecture: arm64
+        include:
+          - xcode: Xcode_15.4
+            os: macos-14
+            architecture: arm64
+          - xcode: Xcode_16
+            os: macos-14
+            architecture: arm64
```
Comment (DivineDominion) [inline .github/workflows/BuildPR.yml:24]: Sorry for asking: do these `include` declarations supersede the 3 lines above that have an array of xcode, os, and architecture? If so, can these 3 lines above be removed?

**Comment by DivineDominion** [comment]
Thank you for powering through this! Just have one question real quick to maybe remove redundant instructions. But we're good to merge either way.

**Review by DivineDominion — APPROVED** [review]

File: .github/workflows/BuildPR.yml (hunk 1 above)
Comment (hannesa2) [reply .github/workflows/BuildPR.yml:24]: No it adds to x86 matrix

**Comment by hannesa2** [comment]
I solved the conflict

File: .github/workflows/BuildPR.yml (hunk 1 above)
Comment (hannesa2) [reply .github/workflows/BuildPR.yml:24]: A normal matrix is a multiplication, so you can't simple add arm64. This must be added to matrix separated

**Comment by DivineDominion** [comment]
Sorry for the delay! With our toddler at home, I just didn't notice that so much time has passed. Thank you for your work on this!

**Pull request merged**